cmd/clusterctl/client/config/providers_client.go

```diff
@@ -111,7 +115,7 @@ func (p *providersClient) defaults() []Provider {
 	// out-of-box experience. This is an opt-in feature; other providers can be added by using the clusterctl configuration file.
 
 	// if you are a developer of a SIG-cluster-lifecycle project, you can send a PR to extend the following list.
-
+        outscaleVersion := GetLastVersion("outscale-dev", "cluster-api-provider-outscale")
```

Comment:
all the other providers use latest in their github url and it automatically resolves to the latest released version. I strongly recommend using the same approach also here